src/index.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import * as serviceWorker from './serviceWorker';
-import {DootRouter} from './router';
+import { DootRouter } from './router';
```

Comment:
How did this get changed?

Reply:
Heres a link explaining this specific change.

[https://www.youtube.com/watch?v=dQw4w9WgXcQ](url)

Reply:
That's the song I imagine eslint and prettier singing to me every time I save a file.